ports/mimxrt/boards/MIMXRT1170_EVK/MIMXRT1170_EVK_flexspi_nor_config.h

```diff
@@ -0,0 +1,263 @@
+/*
```

Comment:
Can this file be put in a common location, like `hal/`?

Reply:
Same as above. Not needed any more.